cps-wallet-connector/README.md

```diff
@@ -0,0 +1,221 @@
+---
+CPS: ?
+Title: Cardano Wallet Connectors
```

Comment:
OK @Ryun1 I'll take the responsibility if doing the wrong thing here, since I'm putting it without "Cardano" on the CPS "Under review" list (I'll leave it to you to change the document itself)...  technically we should have _Cardano_ in all of these titles or none of them 😜

Reply:
Happy to remove "Cardano" -- makes sense.